### ISSUE TITLE
Move bioconda/coreutils to conda-forge/coreutils

### ISSUE THREAD

**Issue by sodre** [conda-forge/coreutils-feedstock#1]
Issue: Move bioconda/coreutils packages to conda-forge/coreutils
<br/>

This is a follow up from @bgruening request during the staged-recipes [PR.](https://github.com/conda-forge/staged-recipes/pull/6144#issuecomment-399254409)

**Comment by sodre** [comment]
@bgruening, did you want to actually move the files at the Anaconda Cloud level or is it enough to just rebuild the 8.25 version?

If the latter, I can't do it directly and we will need someone who has API access to the conda-forge channel to execute:
```
anaconda copy --to-owner conda-forge bioconda/coreutils/8.25
```

**Comment by bgruening** [comment]
@sodre good question. Is the copying possible given the conda-forge guidelines?
I would love to copy it but I see two problems but the recipes and packages are probably not following conda-forge standards. If that is not a problem I can copy them I should have access.

**Comment by isuruf** [comment]
First question, is 8.25 really necesssary? If so, it's best to build it from here. We don't copy packages unless there's a very good reason (bootstrapping ppc64le was one reason)

**Comment by bgruening** [comment]
Yes, reproducibility. Otherwise, we need to keep the old versions in bioconda.

**Comment by sodre** [comment]
@isuruf, @bgruening did you make a decision on how to proceed? I am happy to do the 8.25 version separately.
